examples/dashboard/src/utils/ServerSdk.ts

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+const headers = { "Content-Type": "application/json" };
+
+export class ServerRoomSdk {
```

Comment:
I think this should be in `src/jellyfish`. Would be nice if we could make API of this module somehow similar to what we have in https://github.com/jellyfish-dev/server_sdk_elixir/pull/1?